.github/ISSUE_TEMPLATE/user-story-issue-template.md

```diff
@@ -0,0 +1,21 @@
+---
+name: User Story Issue Template
+about: For user stories
+title: As a _ user, I can _
+labels: type.Story
+assignees: ''
+
+---
+
+so that I can _
```

Comment:
I think `so that` is enough... what does the underscore do?

Reply:
Just for you to fill in :P